### PR TITLE
Add HHVM role to vagrant.yml

### PR DESCRIPTION
**Pull request by hkirsman**

### CODE REVIEW

File: conf/vagrant.yml [hunk 1]
```diff
@@ -14,6 +14,11 @@
    - { role: drupal-db, tags: [ 'drupal-db' ] }
    - { role: letsencrypt, tags: [ 'letsencrypt' ] }
    - { role: sslterminator, tags: [ 'sslterminator' ] }
+# Optional HHVM role.
```
Comment (tharna) [inline conf/vagrant.yml]: Add notification that Wundermachina needs to be pointed to experimental hhvm branch for this to work.

**Comment by hkirsman** [comment]
So you'll merge this particular code to centos 7 branch?

**Comment by tharna** [comment]
Yes, this can be merged to main centos7 branch so people will be aware of the possibility, but it's good to have that additional info on how to use it for the time being until we can have stable release of HHVM with a proper RPM on the main WunderMachina branch.

**Comment by hkirsman** [comment]
Text will get rather long. Maybe add short version there and link to more in-depth documentation?

**Comment by tharna** [comment]
Just add something like: Requires Wundermachina from [branch].
Should be enough :)

**Comment by hkirsman** [comment]
Right, I managed to do a wiki page :)

**Comment by tharna** [comment]
Even better (y)

**Pull request merged**